src/app/components/SocialLogin.jsx

```diff
@@ -12,7 +17,7 @@ export default function SocialLoging() {
           height={28}
           priority
         />
-        Sign Up with Google
+        {`${buttonText} with Google`}
```

Comment:
I think this also works without the ticks
```suggestion
        {buttonText} with Google
```